clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -226,6 +224,41 @@ public void loadExisting() {
     //nothing to do
   }
 
+  private EhcacheEntityResponse invokeLifeCycleOperation(ClientDescriptor clientDescriptor, LifecycleMessage message) {
+    try {
+      switch (message.operation()) {
+        case CONFIGURE: return configure((LifecycleMessage.ConfigureCacheManager) message);
+        case VALIDATE: return validate((ValidateCacheManager) message);
+        case CREATE_SERVER_STORE: return createServerStore(clientDescriptor, (CreateServerStore) message);
+        case VALIDATE_SERVER_STORE: return validateServerStore(clientDescriptor, (ValidateServerStore) message);
+        case RELEASE_SERVER_STORE: return releaseServerStore(clientDescriptor, (ReleaseServerStore) message);
+        case DESTROY_SERVER_STORE: return destroyServerStore(clientDescriptor, (DestroyServerStore) message);
+        default: throw new IllegalArgumentException("Unknown LifeCycle operation " + message);
+      }
+    } catch (Exception e) {
+      return EhcacheEntityResponse.failure(e);
+    }
+  }
+
+  private EhcacheEntityResponse invokeServerStoreOperation(ServerStoreOpMessage message) {
+    try {
+      switch (message.operation()) {
+        case GET: return EhcacheEntityResponse.response(stores.get(message.getCacheId()).get(message.getKey()));
```

Comment:
All the `stores.get` should be wrapped so that they throw a meaningful exception in case there is no such store. Otherwise the caller will just get a useless NPE that happened on the other side of the network ...